README.md

```diff
@@ -33,13 +59,6 @@ reference in app-interface to the commit hash of the changes you made.
 3. Once merged and ready to be promoted to production, update the
 [production environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/0486ef164e70259e5b85c46ab749529238368414/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L39) reference in a similar manner.
 
-## Grafana Dashboards
-
-Refer to the app-interface [instructions](
-https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/monitoring.md#visualization-with-grafana)
-to learn how to develop AppSRE dashboards for RHTAP. This repository serves as
-versioned storage for the [dashboard definitions](dashboards/) and nothing more.
-
```

Comment:
These grafana dashboards are quite new and should not be removed.

Reply:
@amisstea & @yftacherzog Thanks for your inputs. I have added the Grafana dashboards back and have also added GitLab links for the alerting rules.